ZenPacks/zenoss/ZenPackLib/zenpacklib.py

```diff
@@ -27,7 +27,7 @@
 """
 
 # PEP-396 version. (https://www.python.org/dev/peps/pep-0396/)
-__version__ = "2.0.9"
+__version__ = "2.0.10"
```

Comment:
Version bump should be made directly on the hotfix/2.0.10 branch before this commit.